functions/internal/javascript/gram-start.mjs

```diff
@@ -194,10 +194,7 @@ async function writeHTTPResponse(pipeFile, response) {
     return;
   }
 
-  const dest = pipeFile.createWriteStream({
-    autoClose: true,
-    emitClose: true,
-  });
+  const dest = pipeFile.createWriteStream();
```

Comment:
these options were not needed given we are closing the named pipe at the end of a function invocation.